packages/vite/src/node/optimizer/scan.ts

```diff
@@ -41,6 +41,7 @@ export const importsRE =
   /(?<!\/\/.*)(?<=^|;|\*\/)\s*import(?!\s+type)(?:[\w*{}\n\r\t, ]+from\s*)?\s*("[^"]+"|'[^']+')\s*(?=$|;|\/\/|\/\*)/gm
 
 const multilineCommentsRE = /\/\*(.|[\r\n])*?\*\//gm
+const singleCommentsRegex = /\/\/.*/g
```

Comment:
Please use the `RE` suffix as everywhere else in the project

Reply:
Done.